### PR TITLE
Add status invite

### PR DESCRIPTION
**Pull request by viniciusreiss**
close issue criar botão para cancelar invite #85

### CODE REVIEW

File: src/containers/Contractor/List/index.js [hunk 1]
```diff
@@ -54,14 +54,17 @@ class ContractorList extends Component {
...
       dataIndex: 'cancel',
       width: '15%',
-      render: cancel => {
+      render: (cancel, invite) => {
         const {deleteInvite} = this.props
-        console.log(deleteInvite)
-        return (
-          <Button onClick={deleteInvite}
-          >
+        const status = invite.status
```
Comment (alexandrejuk) [inline src/containers/Contractor/List/index.js]: You should use destructure. example const { status, id } =  invite

File: src/pages/Logged/Contractor/List/index.js [hunk 2]
```diff
@@ -2,36 +2,47 @@ import React, { Component } from 'react'
...
 
-  componentDidMount() {
+   componentDidMount() {
     this.contractorService = new ContractorService(
       this.props.accountId,
       this.props.companyId
     )
-
-    this.getContractors()
+  this.getContractors()
```
Comment (vitorsilvalima) [inline src/pages/Logged/Contractor/List/index.js]: you should indent here correctly

File: src/pages/Logged/Contractor/List/index.js [hunk 3]
```diff
@@ -2,36 +2,47 @@ import React, { Component } from 'react'
...
       const { data: dataValues } = await this.contractorService.getContractors()
-      this.setState({ dataValues, loading: false })  
-    }
+      this.setState({ dataValues, loading: false }) 
+    })
   }
+  
+   deleteInvite = async (id) => { 
+     await this.contractorService.deleteById(id)
+    this.getContractors()
```
Comment (vitorsilvalima) [inline src/pages/Logged/Contractor/List/index.js]: you should indent here correctly

File: src/containers/Contractor/List/index.js (hunk 1 above)
Comment (viniciusreiss) [reply src/containers/Contractor/List/index.js]: Ok, Solved.

**Pull request merged**